IDE/Renesas/e2studio/DK-S7G2/user_settings.h

```diff
@@ -0,0 +1,77 @@
+
+#ifndef USER_SETTINGS_H
+#define USER_SETTINGS_H
+
+//#define DEBUG_WOLFSSL
+
+#define NO_MAIN_DRIVER
+#define USE_CERT_BUFFERS_2048
+#define USE_CERT_BUFFERS_256
+
+/* print out cycles per byte with benchmark when component r_wdt WDT is enabled */
+#define SYNERGY_CYCLE_COUNT
+#define BENCH_EMBEDDED
+
+/* Use turn on all SCE acceleration */
+#define WOLFSSL_SCE
+
+/* Used to turn off TRNG */
+#define WOLFSSL_SCE_NO_TRNG
```

Comment:
Do we want to default to disabling HW crypto here, or should we default to having it enabled?